jenkinsci/tap-plugin
# [JENKINS-74247] extract inline JS script in TapStreamResult/body.jelly

[JENKINS-74247](https://issues.jenkins.io/browse/JENKINS-74247)

**WHY**
https://www.jenkins.io/doc/developer/security/csp/

### Testing done

To find the view:

1. create freestyle project and add execute shell step with the below:
```
cat > test.tap <<EOF
TAP version 13
1..3
ok 1 - Basic test execution
not ok 2 - Failed test case
  ---
  message: Test failed as expected
  severity: fail
  ...
ok 3 - Final test # SKIP Skipping for demo
EOF
```

2. Add POST-Build Action: Publish TAP results and set the test Results input to : `test.tap`

_______

BEFORE CHANGES:

https://github.com/user-attachments/assets/26b09c9a-b8a9-41a3-9b6e-f25ae0f439ba


AFTER CHANGES:

https://github.com/user-attachments/assets/1d684d45-bdce-4df5-b800-5b0d338162cd





### Submitter checklist
- [] Ensure you have provided tests - that demonstrates feature works or fixes the issue
